server/ERNI.PhotoDatabase.Annotator/YoloParser/YoloBoundingBox.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System.Drawing;
+
+namespace ERNI.PhotoDatabase.Annotator.YoloParser
+{
+    public class BoundingBoxDimensions : Dimensions { }
```

Comment:
Is it necessary to introduce a new calss for this?

Reply:
removed old unused class